__tests__/integration/configuration/docker-compose.yml

```diff
@@ -5,16 +5,18 @@ services:
   # https://github.com/HathorNetwork/rfcs/blob/master/text/0033-private-network-guide.md
 
   fullnode:
-    image: hathornetwork/hathor-core
+    image:
+      ${HATHOR_LIB_INTEGRATION_TESTS_FULLNODE_IMAGE:-hathornetwork/hathor-core:experimental-nano-sdk-v1.1}
```

Comment:
If I changed the image only in the CI environment the tests would fail in the development environment, so the devs would need to change the env variable manually as well.

Apart from that, there's a new hathor-core parameter that is needed for the nano tests. I didn't see any downside of having it like this, apart from the need to change it back when the nano code becomes available in the latest docker image.